application-xpoll-ui/src/main/resources/XPoll/XPollSheet.xml

```diff
@@ -116,7 +147,7 @@
   #end
 #end
 ## Flag to know wheter the options must be displayed as checkbox or radio inputs.
-#set ($isRadio = $pollObj.getProperty('type').value == 1)
+#set ($isRadio = $pollObj.getProperty('type').value == 'single')
```

Comment:
I think we need to handle existing polls that have the type set to 0 or 1 no?

* either check for both ``1`` and "single" (and equivalent for ``multi``)
* or write a migration script for the existing polls